src/Page/Donate/styles.css

```
@@ -0,0 +1,292 @@
+
+.donate-page {
+  background-color: #fff;
+  text-align: center;
+}
+.donate-page .page-content {
+  width: 100%;
+}
+
+.partition-line {
+  border-top: 2px solid #cdcdcd;
+}
+.donate-page-section {
+  margin-bottom: 64px;
+}
+.donate-page-image-section {
+  margin: 40px 0;
+  background-image: url("./assets/bg.png");
+  height: 800px;
+  width: 100%;
+  background-position: center;
+  background-repeat: none;
+  background-size: cover;
+  position: relative;
+}
+.middleImg {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+.donation-impact-data-2 {
+  margin-top: 200px !important;
+}
+.section-donate-title {
+  margin-top: 80px;
+  line-height: 42px;
+}
+
+.donation-details-container {
+  text-align: left;
+}
+
+.ac-details {
+  font-weight: 500;
+}
+.donation-details-container ul li {
+  white-space: nowrap;
+  margin-bottom: 10px;
+}
+.donate-liner {
+  border: 1px solid #cdcdcd;
+  background-color: #cdcdcd;
+  width: 400px;
+  align-items: center;
+  margin-left: 10px auto;
```

Did you mean `margin` here?
```suggestion
  margin: 10px auto;
```